src/elektra/redshift.ini

```diff
@@ -0,0 +1,308 @@
+default = continual
+example = oneshot
+opt/long = mode
+opt/arg = required
+
+[verbose]
+type = boolean
+description = Verbose output.
+default = 0
+example = 1
+opt = v
+opt/arg = none
+
+[version]
+type = boolean
+description = Show program version.
+default = 0
+example = 1
+opt = V
+opt/arg = none
+
+[temp/day]
+type = unsigned_short
+description = The color temperature the screen should have during daytime.
+default = 6500
+example = 6500
+check/type = unsigned_short
+check/range = 1000-25000
+opt/long = temp-day
+opt/arg = required
+
+[temp/night]
+type = unsigned_short
+description = The color temperature the screen should have during nighttime.
+default = 4500
+example = 4500
+check/type = unsigned_short
+check/range = 1000-25000
+opt/long = temp-night
+opt/arg = required
+
+[temp/oneshotmanual]
+type = unsigned_short
+description = The color temperature the screen should have when oneshotmanual mode is used.
+default = 6500
+example = 6500
+check/type = unsigned_short
+check/range = 1000-25000
+opt/long = temp-oneshotmanual
+opt/arg = required
+
+[fade]
+type = boolean
+description = Whether transitions between color temperatures (e.g. from daytime to nighttime) should happen fast or slow.
+default = 1
+example = 0
+opt = f
+opt/long = fade
+opt/arg = none
+
+[brightness/day]
+type = float
+description = The screen brightness during daytime. If both day and night brightness are set, these will overrule the value of brightness.
+default = 1.0
+example = 0.8
+check/type = float
+check/range = 0-1
+opt/long = brightness-day
+opt/arg = required
+
+[brightness/night]
+type = float
+description = The screen brightness during nighttime. If both day and night brightness are set, these will overrule the value of brightness.
+default = 1.0
+example = 0.8
+check/type = float
+check/range = 0-1
+opt/long = brightness-night
+opt/arg = required
+
+[gamma/day]
+type = string
+description = The screen gamma during daytime. Supported formats: 1. One value, that will be used for red, green and blue (e.g. 0.9). 2. Three colon-separated values for red, green and blue respectively (e.g. 0.9:0.9:0.9).
+default = 1.0:1.0:1.0
+example = 0.9
+; Regex ensures format "float" or "float:float:float". Adapted from https://www.regular-expressions.info/floatingpoint.html
+check/validation = ^([0-9]*[\.,]?[0-9]+)(:([0-9]*[\.,]?[0-9]+):([0-9]*[\.,]?[0-9]+))?$
+opt/long = gamma-day
+opt/arg = required
+
+[gamma/night]
+type = string
+description = The screen gamma during nighttime. Supported formats: 1. One value, that will be used for red, green and blue (e.g. 0.9). 2. Three colon-separated values for red, green and blue respectively (e.g. 0.9:0.9:0.9).
+default = 1.0:1.0:1.0
+example = 0.9
+; Regex ensures format "float" or "float:float:float". Adapted from https://www.regular-expressions.info/floatingpoint.html
+check/validation = ^([0-9]*[\.,]?[0-9]+)(:([0-9]*[\.,]?[0-9]+):([0-9]*[\.,]?[0-9]+))?$
```

Comment:
Here a check/validation/message is appropriate (that "float" or "float:float:float" is expected).

Reply:
Thanks, I added a check/validation/message for all regexes!